client/app/account/edit/component/EditAccountForm.tsx

```diff
@@ -0,0 +1,127 @@
+import { MdAccountCircle } from 'react-icons/md'
+import { useAccountEdit } from '@/(auth)/hooks/useAccountEdit'
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+  Input,
+  Button,
+} from '@/_components/ui'
+import { colorMap } from '@/_consts'
+import { UserType } from '@/_types'
+
+type EditAccountFormProps = {
+  user: UserType
+}
+const EditAccountForm = ({ user }: EditAccountFormProps) => {
+  const { form, onSubmit, isPending } = useAccountEdit(user)
+
+  const colorArray = Object.entries(colorMap).map((colorArray) => {
+    return {
+      value: colorArray[0],
+      color: colorArray[1],
+    }
+  })
+
+  const iconWatch = form.watch('icon')
+
+  return (
+    <Form {...form}>
+      <form onSubmit={onSubmit} className='mx-auto mt-8 w-full'>
+        <div className='space-y-5'>
+          <FormField
+            control={form.control}
+            name='icon'
+            render={() => (
+              <FormItem>
+                <FormControl>
+                  <div className='flex'>
+                    <div className='flex h-full columns-1 flex-col'>
+                      <FormLabel className=' font-semibold'>Icon</FormLabel>
+                      <MdAccountCircle
+                        className={`mt-12 text-4xl ${colorMap[user.icon as keyof typeof colorMap]}`}
+                        size={52}
+                      />
+                    </div>
+                    <div className='flex w-full columns-11 items-center justify-center'>
+                      <div className='grid grid-cols-3 items-center justify-center gap-4'>
+                        {colorArray.map((colorValue) => {
+                          return (
+                            <FormItem
+                              className='flex items-center space-x-2 space-y-0'
+                              key={colorValue.value}
+                            >
+                              <FormLabel
+                                className={`cursor-pointer rounded-full border border-primary ring-offset-background ${iconWatch === colorValue.value && 'ring-2 ring-ring ring-offset-2 '}`}
+                                onClick={() =>
+                                  form.setValue('icon', colorValue.value)
+                                }
+                              >
```

Comment:
2 things! 

1. 
For the accessibility, I want you to add aria label.
```.js
  aria-label={`Select ${colorValue.value} color`}
```
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

2. 
Sorry I put the border but simple is better looking. So could you change like this if you also think it's better?

```.js
  className={`cursor-pointer rounded-full ring-offset-background ${iconWatch === colorValue.value && 'ring-2 ring-ring ring-offset-0'}`}

```

<img width="212" alt="Screenshot 2024-03-21 at 1 35 55 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/f216f122-d2f2-4a37-ad76-e2b47a3f6cf9">

Reply:
Fixed it. 3140d08d3db1079358f86b49eea7a1e1042cf4db